docker/proxy/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM birgerk/apache-letsencrypt
+
+
+#RUN apt-get update && \
+#    apt-get -f stretch install libapache2-mod-auth-openidc && \
+#    rm -rf /var/lib/apt/lists/*
+
```

Comment:
you should install the tools you need as part of the build. I don't think it will make much difference in the build time or the overall size